client/src/app/slides/mediafiles/mediafile/mediafile-slide.component.ts

```diff
@@ -12,4 +14,8 @@ export class MediafileSlideComponent extends BaseSlideComponent<MediafileSlideDa
     public constructor() {
         super();
     }
+
+    @computed get url() {
```

Comment:
@boehlke For what case is the `@computed` there?

Reply:
It is used in the componten.html...

Reply:
Is the computed necessary there?

Reply:
Hi Finn, I am not sure... please decide for youself... I am not very friends with angular.
I can delete it if you want.